model/src/main/java/org/mskcc/cmo/metadb/model/NormalSampleEntity.java

```diff
@@ -0,0 +1,190 @@
+package org.mskcc.cmo.metadb.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.mskcc.cmo.shared.Library;
+import org.mskcc.cmo.shared.QcReport;
+import org.mskcc.cmo.shared.SampleManifest;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+import org.neo4j.ogm.id.UuidStrategy;
+import org.neo4j.ogm.typeconversion.UuidStringConverter;
+
+@NodeEntity(label = "cmo_metadb_sample_metadata")
+public class NormalSampleEntity extends SampleManifest {
+    @Id @GeneratedValue(strategy = UuidStrategy.class)
+    @Convert(UuidStringConverter.class)
+    private UUID uuid;
+    @Relationship(type = "SP_TO_SP", direction = Relationship.INCOMING)
+    private List<Sample> sampleList;
+    @Relationship(type = "PX_TO_NORMAL", direction = Relationship.INCOMING)
+    private PatientMetadata patient;
+    @Relationship(type = "SAMPLE_MANIFEST", direction = Relationship.OUTGOING)
+    private SampleManifestJsonEntity sampleManifestJsonEntity;
+
+    public NormalSampleEntity() {
+        super();
+    }
+
+    /**
+     * SampleManifestEntity constructor.
```

Comment:
comments should be updated (replace SampleManifestEntity with the name of this class)